components/purchase/PurchaseButtons.tsx

```diff
@@ -4,9 +4,27 @@ import { useTranslation } from "next-i18next"
 const PurchaseButtons: React.FC = () => {
   const { t } = useTranslation("booklet")
   return (
-    <div>
-      <button onClick={() => window.open("https://google.com", "_blank")}>{t("buttons.usa")}</button>
-      <button onClick={() => window.open("https://google.com", "_blank")}>{t("buttons.international")}</button>
+    <div className="d-flex flex-column flex-md-row justify-content-center">
+      <button onClick={() => window.open("https://www.davidsonpublishing.org/prayer-guides1.html", "_blank")}>
+        {t("buttons.northAmerica")}
+      </button>
+      <button
+        onClick={() =>
+          window.open(
+            "https://docs.google.com/forms/d/e/1FAIpQLSf03r2GXDfFa17f5ICL_HTy_NuQOpaJcmNgRyFQN10ghgEYqQ/viewform",
+            "_blank"
```

Comment:
Just out of curiosity, what is the "_blank" as the second argument of window.open do?